auth/src/main/java/org/robojackets/apiary/auth/ui/permissions/MissingHiddenTeamsCallout.kt

```diff
@@ -0,0 +1,25 @@
+package org.robojackets.apiary.auth.ui.permissions
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.OutlinedButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import org.robojackets.apiary.base.ui.callout.WarningCallout
+
+@Composable
+fun MissingHiddenTeamsCallout(onRefreshTeams: () -> Unit) {
```

Comment:
Spacing is still wonky...idk what the correct CSS term for it is but there should be more space between the edge of the container and the text. There is also uneven spacing around the left side vs below the button.